### PR TITLE
Nuzhy/86897/eu social signup redirection notice

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by NikitK-deriv — COMMENTED** [review]
Could you please check?
https://user-images.githubusercontent.com/103182473/226820101-3b0b091e-63b7-47dd-9ce5-3b6f5ad9f003.mov
And please advice where and how to check behaviour with notice

**Comment by Nuzhy-Deriv** [comment]
> Could you please check? https://user-images.githubusercontent.com/103182473/226820101-3b0b091e-63b7-47dd-9ce5-3b6f5ad9f003.mov And please advice where and how to check behaviour with notice

**Review by prince-deriv — APPROVED** [review]

**Comment by behnaz-deriv** [comment]
Thank for adding changes to [src/pages/landing/ebooks/components/_get-ebook.tsx](https://github.com/binary-com/deriv-com/pull/4371/files#diff-d67e0a4d890aa8847b54cf83ff128ff052db3badc22fd895f67c5a8cc43ed0ae) file, LGTM :pray:

File: src/components/custom/_signup-new.tsx [hunk 1]
```diff
@@ -218,6 +233,41 @@ const SignupNew = ({
...
+    const [social_url, setSocialURL] = useState({ google: '', facebook: '', apple: '' })
+    useEffect(() => {
+        if (isBrowser()) {
+            setSocialURL({
+                google: Login.socialLoginUrl('google'),
+                facebook: Login.socialLoginUrl('facebook'),
+                apple: Login.socialLoginUrl('apple'),
+            })
+        }
+    }, [isBrowser()])
```
Comment (prince-deriv) [inline src/components/custom/_signup-new.tsx:245]:
```suggestion
    }, [isBrowser])
```

File: src/translations/ach.json [hunk 2]
```diff
@@ -3204,4 +3204,4 @@
   "-1773685845": "crwdns1781049:0crwdne1781049:0",
   "-661813412": "crwdns1781051:0crwdne1781051:0",
   "-1520902282": "crwdns1781053:0crwdne1781053:0"
-}
\ No newline at end of file
+}
```
Comment (prince-deriv) [inline src/translations/ach.json:3207]: Can we exclude this file?

Comment (Nuzhy-Deriv) [reply src/translations/ach.json:3207]: It was auto picked up. Let me see after solving merge conflict.

**Review by habib-deriv — APPROVED** [review]
LGTM

**Comment by Nuzhy-Deriv** [comment]
Closing this PR since these changes are not required.
https://deriv-group.slack.com/archives/C049GHXQV7Z/p1680593920091489?thread_ts=1677828619.885439&cid=C049GHXQV7Z

**Pull request closed without merging**